src/app/components/collectibleCollage/collectibleCollage.tsx

```diff
@@ -1,4 +1,7 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
should we turn off this eslint rule globally? 😛

Reply:
would very much like to do so 😅